README-developers.md

```diff
@@ -1,24 +1,42 @@
 # Eel Developers
 
-## Setting up a development environment
+## Setting up your environment
 
 In order to start developing with Eel you'll need to checkout the code, set up a development and testing environment, and check that everything is in order.
 
-Clone the repository:
+### Clone the repository:
```

Comment:
There's a hanging over `:` that can probably be dropped here :)

Reply:
Fixed, thanks!